sphinx/templates/latex/sphinxmessage.sty_t

```diff
@@ -0,0 +1,6 @@
+<%= babel(renewcommand('\literalblockcontinuedname', escape(_('continued from previous page')))) %>
```

Comment:
It would be much better if we can move `babel(...)` to TeX macro...

Reply:
not sure I understand what you mean here

Reply:
what about `sphinxcaptions.sty` as filename ? or `sphinxlocalizedstrings.sty`? or perhaps "message" is familiar from gettext etc...

Reply:
ah, sorry. I think representing this code by LaTeX is better and simple:
```
\addtocaptions{\renewcommand{\literalblockcontinuedname}{<%= _('continued from previous page' | e %>}}
```

Reply:
That's good news. We've not supported multilingual documents. So it would not be needed at least now.
+1 for removing :-)

Reply:
+1 also for directly using `\renewcommand`'s in the message file for all those `\sphinx...name` macros, and things like `\autopagerefname`. Here is the list of things needing `\addto\captions...` when babel (or I guess polyglossia) is loaded:

```latex
\prefacename\refname\abstractname\bibname\chaptername\appendixname
\contentsname\listfigurename\listtablename\indexname\figurename
\tablename\partname\enclname\ccname\headtoname\pagename\seename
\alsoname\proofname\glossaryname
```